### ISSUE TITLE
'torch.no_grad()' in 'calc_policy_loss()'

### ISSUE THREAD

**Issue by Simcs** [ku2482/sac-discrete.pytorch#24]
Hi, I was referring to this repo and another repo at the link below.

https://github.com/p-christ/Deep-Reinforcement-Learning-Algorithms-with-PyTorch/blob/master/agents/actor_critic_agents/SAC_Discrete.py

I think this is the reimplemented version, but there seems to be a little difference in 'calc_policy_loss()' function.

In this version, you seem to add 'torch.no_grad' when forwarding online_critic, so I wondered if there was a reason.

Thanks in advance!

**Comment by toshikwa** [comment]
Sorry for the late response.

Because that part is independent of the policy, so the gradient of `policy_loss` doesn't depend on it.
Does it make sense?
